### PR TITLE
Add GCB as CI

### PR DESCRIPTION
**Pull request by wenovus**

### CODE REVIEW

**Comment by wenovus** [comment]
Sorry I forgot to make it a draft at first @aashaikh , the PR is ready for review now.

**Comment by wenovus** [comment]
Please comment as to which validators to keep in the status, and which validators to have in the compatibility report.

**Comment by robshakir** [comment]
@wenovus - it looks like a few of these are failing due to missing modules (`iana-if-type` for example). Could you confirm why that is?

I think the current breakdown of compatibility vs. checks makes sense. We need to push the private repo to this one to ensure that the upstream fixes are also here.

**Comment by wenovus** [comment]
@robshakir I fixed the bug with ConfD. I believe the rest of the issues has to do with a divergence with the `models` repo.

**Comment by aashaikh** [comment]
sorry for the delay on this - I will publish model updates to the public repo today/tonight to resolve.

**Comment by wenovus** [comment]
Np, anytime is fine.

**Comment by aashaikh** [comment]
Wen, FYI, have pushed fixes to public -- hopefully that will unblock this.

**Comment by wenovus** [comment]
Thanks it works now! I've enabled CI for pushes to master, so hopefully that will succeed and update the badges.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
...
+[![pyang@1.7.8](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyang@1.7.8.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyang@1.7.8.html)
+[![pyangbind](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyangbind.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:pyangbind.html)
+[![goyang/ygot](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:goyang-ygot.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:goyang-ygot.html)
+[![yanglint](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:yanglint.svg)](https://storage.googleapis.com/artifacts.disco-idea-817.appspot.com/compatibility-badges/openconfig-public:yanglint.html)
+
```
Comment (aashaikh) [inline README.md:6]: Per discussion in the working group meeting, might be nice to include a visual indicator on the status of the OpenConfig linter as well.   Not a blocker, but a nice-to-have IMO.

Comment (wenovus) [reply README.md:6]: This is [in the works](https://github.com/openconfig/models/pull/855/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8) in the models repo. I will add it to public once it's in models.

Comment (wenovus) [reply README.md:7]: The badge is working now.

**Pull request merged**